dev-docs/bidders/nexverse.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: bidder
+title: Nexverse
+description: Prebid Nexverse Bidder Adapter
+biddercode: nexverse
+aliasCode: nexverse
+tcfeu_supported: true
+dsa_supported: true
+gvl_id: none
```

Comment:
gvl_id is strictly required if you claim tcfeu_supported: true. Either add the number here or change tcfeu_supported to false.